src/index.js

```diff
@@ -80,6 +101,9 @@ function startQtumProcess(reindex) {
   qtumProcess.on('close', (code) => {
     logger.debug(`qtumd exited with code ${code}`);
   });
+
+  // repeatedly check if qtumd is running
+  checkInterval = setInterval(checkQtumd, 1000);
```

Comment:
will `setTimeout` be easier? No need to cleanInterval

Reply:
`setInterval` is for repeating calls. Wanted to make sure qtumd is running fully before launching all the services.

Reply:
@dwalintukan this line is a recursive call to start itself so dunt need repeating calls.

Reply:
It's not recursive. The `checkInterval` is holding the interval timer itself and `checkQtumd` only has logic to clear the interval.